### PR TITLE
Enable build cache

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**

**Purpose**

### CODE REVIEW

File: src/web_app/js/infobox.js [hunk 1]
```diff
@@ -133,10 +133,10 @@ InfoBox.prototype.updateInfoDiv = function() {
         }
       }
       contents += this.buildLine_('LocalAddr', localAddr +
-          ' (' + localAddrType + (localTypePref !== undefined ?
-          ' ' + formatTypePreference(localTypePref) : '') + ')');
-      contents += this.buildLine_('RemoteAddr', remoteAddr +
-          ' (' + remoteAddrType + ')');
+          ' (' + localAddrType + (localTypePref !== undefined ? '' +
```
Comment (MirkoBonadei) [inline src/web_app/js/infobox.js]: This seems not related to this pull request.

Comment (KaptenJansson) [inline src/web_app/js/infobox.js]: I've fixed that in #467 but I got blocked testing the cache on this lint error.

**Comment by KaptenJansson** [comment]
I will rebase this once I've fixed everything. Cloud engine install logic is not happy with the caching for some reason....
